pkg/trace/flags/flags_windows.go

```diff
@@ -1,31 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-2019 Datadog, Inc.
-
-// +build windows
-
-package flags
-
-import (
-	"flag"
-	"path/filepath"
-
-	"github.com/DataDog/datadog-agent/pkg/util/winutil"
-)
-
-// DefaultConfigPath specifies the default configuration path.
-var DefaultConfigPath = "c:\\programdata\\datadog\\datadog.yaml"
```

Comment:
Not sure how do we pass this as default config. What is the new default config path for Windows?

Reply:
Same here, just search the page, you'll find it.